guides/doc-Monitoring_Project/topics/proc_installing-pcp-packages.adoc

```diff
@@ -0,0 +1,35 @@
+[id='installing-pcp-packages_{context}']
+= Installing PCP Packages
+
+This procedure describes how to install the PCP packages.
+
+.Prerequisites
+
+* Ensure you have a minimum of 20 GB space available in the `/var/log/pcp` directory.
++
+The default PCP data retention policy is to retain only that data collected during the past 14 days. Data storage per day is estimated to use usually between 100 MB and 500 MB of disk space, but may use up to several gigabytes.
```

Comment:
```suggestion
PCP's default data retention policy is to retain data collected within the last 14 days.
Data storage per day is estimated to use usually between 100 MB and 500 MB of disk space, but it can go upto several gigabytes.
```

Reply:
Tried to make it straight.